.release/VERSION

```diff
@@ -0,0 +1 @@
+0.0.0
```

Comment:
TODO: Update to match current version of Consul when PR is ready for full review.